### PR TITLE
Feature/create user

### PR DESCRIPTION
**Pull request by adrianfernandezdz**
feat: create-user components created and styled. Added into the home view. Instantiated it into the Home view.

### CODE REVIEW

File: src/components/create-user.vue [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    <button :disabled="!isFormFilled" type="submit" v-on:click="create" value="Create User">Register</button>
+  </div>
+</template>
+
+<script>
+import { validateMail } from "@/utils.js";
+
+export default {
+  name: "CreateUser",
+  props: {},
```
Comment (edu-aguilar) [inline src/components/create-user.vue]: u dont need props if it's empty

File: src/components/create-user.vue [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    return {
+      mail: "mailexample@gmail.com",
+      name: "nameexample",
+      password: "passwordexample",
+      birthdate: ""
+    };
+  },
+  computed: {
+    isFormFilled: function() {
+      return (
```
Comment (edu-aguilar) [inline src/components/create-user.vue]: u dont need these parenthesis

File: src/components/create-user.vue [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        validateMail(this.mail) && this.name && this.password && this.birthdate
```
Comment (edu-aguilar) [inline src/components/create-user.vue]: can user add a future date? just saying.

File: src/components/create-user.vue [hunk 4]
```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="login-wrapper">
```
Comment (edu-aguilar) [inline src/components/create-user.vue]: CPOP (copy paste oriented programming) careful bro! this is not a login like my component!

**Comment by edu-aguilar** [comment]
Conflict resolved =)

File: src/components/create-user.vue (hunk 1 above)
Comment (adrianfernandezdz) [reply src/components/create-user.vue]: Fixed my bro!

File: src/components/create-user.vue (hunk 2 above)
Comment (adrianfernandezdz) [reply src/components/create-user.vue]: which ones?

File: src/components/create-user.vue (hunk 3 above)
Comment (adrianfernandezdz) [reply src/components/create-user.vue]: MMmmh, you are right!

Need to think a way to validate it before ending this UH.

File: src/components/create-user.vue (hunk 4 above)
Comment (adrianfernandezdz) [reply src/components/create-user.vue]: fixed my bro!

**Pull request merged**